### PR TITLE
[videodb] Add support for overview (plot) in movie sets.

### PR DESCRIPTION
**Pull request by ghost**
Makes it possible to specify an overview/plot in movies sets.
Skins can retrieve it via ListItem.Plot (in fact Confluence doesn't need any
modification - "set overview" will be shown where the plot for movies is).

This changes the format of the `<set>` tag for movie NFO files.
Current:

``` xml
<set>My movie set</set>
```

Proposed:

``` xml
<set>
  <name>My movie set</name>
  <overview>This is my awesome movie set.</overview>
</set>
```

The `<overview>` tag is optional, and the current format will still be supported
to not break existing scrapers or local NFO files.

See:
- http://forum.kodi.tv/showthread.php?tid=237761
- https://www.themoviedb.org/collection/121938-the-hobbit-collection
  (API does provide the "overview".)

### CODE REVIEW

**Comment by Montellese** [comment]
Looks good apart from the minors.

**Comment by uNiversaI** [comment]
@anaconda see http://forum.kodi.tv/showthread.php?tid=181318 Will link this PR there since its related.

**Comment by anaconda** [comment]
Thanks for your review. I have add 2 fixup commits to address your comments.

**Comment by kfarhat** [comment]
Please do not send me any email

**Comment by anaconda** [comment]
> Please do not send me any email
unsubscribe.
unwatch the repo.
fix for notification settings.

**Comment by DanCooper** [comment]
It's also possible to add an additional node <sortmethod> to predefine "sort by year" or "sort by title" (year = 0 / title = 1)?

File: xbmc/video/VideoDatabase.cpp [hunk 1]
```diff
@@ -4602,11 +4631,16 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE tvshow ADD userrating integer");
     m_pDS->exec("ALTER TABLE musicvideo ADD userrating integer");
   }
+
+  if (iVersion < 97)
+  {
```
Comment (Montellese) [inline xbmc/video/VideoDatabase.cpp]: No curly braces

**Comment by Montellese** [comment]
Looks good apart from the minor.

**Comment by Montellese** [comment]
jenkins build and merge

**Pull request merged**